Add Read You

th0mcat opened the pull request:
No issue to close, but adding info about newish Android app Read You.
  * https://github.com/Ashinch/ReadYou
  * **Note: GReader/Fever support is coming soon**, [here's](https://github.com/Ashinch/ReadYou/pull/185) the PR, [this branch](https://github.com/Ashinch/ReadYou/tree/feature/accounts-settings) has the front-end done.

PR should probably just be held open until the support has been fully added.

th0mcat committed:
Add Read You

th0mcat committed:
Update README.fr.md

th0mcat committed:
Add Read You

th0mcat committed:
Add Read You

th0mcat committed:
Add Read You

th0mcat committed:
Add Read You

Alkarex commented:
Hello,
I have just merged another PR making some conflicts. Can you please fix them (the easiest is to re-add your line in the two readmes).
Please ping us when there is something new in ReadYou. I have posted some tips https://github.com/Ashinch/ReadYou/issues/221

math-GH commented:
ping @th0mcat

Alkarex commented:
Alpha version https://github.com/Ashinch/ReadYou/discussions/542

Alkarex committed:
Merge branch 'edge' into pr/th0mcat/4633

Alkarex committed:
Beta version ready
https://github.com/Ashinch/ReadYou/pull/536
https://github.com/Ashinch/ReadYou/discussions/478
https://github.com/Ashinch/ReadYou/discussions/542
